storybook/src/stories/Form/Buttons/FilterButton.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {IconSVGs} from '@future-standard/icons';
+import { text, select, boolean } from '@storybook/addon-knobs';
+import { FilterButton } from 'scorer-ui-kit';
+import styled from 'styled-components';
+
+export default {
+  title: 'Form/Buttons',
```

Comment:
Could we move this to the `Filters` section instead of having it in `Form` please.